src/components/Visualizations/Table/Sparkline.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  YAxis,
+  BarChart,
+  Bar,
+  Cell,
+  ReferenceLine,
+  Tooltip,
+} from 'recharts';
+import { StyledContainer, SparklineContainer } from './Styled';
+import { StyledCustomizedTooltip } from '../../Visualizations/Graph/Styled';
+import constants from '../../constants';
+
+const CustomizedTooltip = ({ label, external }) => (
+  <StyledCustomizedTooltip>
+    <div className="label">{label} - {label + 1}</div>
+    {external && external[label] && (
+      <div>
+        <div>
+          CS This Minute: {external[label].delta}
```

Comment:
These strings should be localized

Reply:
I'd also recommend indicating which minute it was on the tooltip

Reply:
The heading in the tooltip box is the minute number (range really).  I can expand this somewhat, but it starts to get a bit cumbersome